spring-integration-file/src/main/java/org/springframework/integration/file/remote/AbstractFileInfo.java

```diff
@@ -57,10 +56,9 @@ public String toJson() {
 
 	@Override
 	public String toString() {
-		return "FileInfo [isDirectory=" + isDirectory() + ", isLink=" + isLink()
-				+ ", Size=" + getSize() + ", ModifiedTime="
-				+ new Date(getModified()) + ", Filename=" + getFilename()
-				+ ", RemoteDirectory=" + getRemoteDirectory() + ", Permissions=" + getPermissions() + "]";
+		return "FileInfo [isLink=" + isLink()
+				+ ", Filename=" + getFilename()
+				+ ", RemoteDirectory=" + getRemoteDirectory() + "]";
```

Comment:
As discussed out of bounds; it would be better to leave this as-is and overload in any subclass that needs this change.